test/rules/typedef/variable-declaration-ignore-function/test.ts.lint

```diff
@@ -0,0 +1,17 @@
+// Should all pass
+var foo = function(): void {};
```

Comment:
There should be some legitimate lint complaints in this file to test that the existing rule logic for cases other than variable-declared functions still fires.